src/PullCountStackTooltip.css

```diff
@@ -0,0 +1,12 @@
+.rh-pull-count-stack-tooltip {
```

Comment:
Do you think you can redefine these styles using patternfly tokens? That may mean defining the styles inline with the component. The reason is I've been trying to keep styles as generic and customizable as possible so that a user of this web component can use their own custom patternfly CSS variables.

Reply:
Replaced what could with react-tokens.